CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2021-29-12
+
+### Breaking Changes
+
+- Support only NodeJS version 16.x.x
```

Comment:
Thanks jerome for this upgrade !
So if anyone use this version with 14.x or 12.x, poolifier is not going to work ? Is that accurate?

Reply:
For cluster pool, yes.